### ISSUE TITLE
I don't get DDIMInverseScheduler working

### ISSUE THREAD

**Issue by kaibioinfo** [huggingface/diffusers#2577]
### Describe the bug

I want to try the  StableDiffusionPix2PixZeroPipeline with Stable Diffusion v2.1. However, when I run the code from the example (https://huggingface.co/docs/diffusers/api/pipelines/stable_diffusion/pix2pix_zero) I just get noisy images back. 

So I tried something simpler and just want to get the DDIMInverseScheduler running. Again, I just get strange images back. I'm a total newbie in stable diffusion, so I hope this is not a silly question.

Here is the minimal code example that is already not working:

### Reproduction

```python
from diffusers import StableDiffusionPipeline, DDIMScheduler, DDIMInverseScheduler, StableDiffusionPix2PixZeroPipeline
import torch

# load models
model_id = "stabilityai/stable-diffusion-2-1"
pipe = StableDiffusionPipeline.from_pretrained(model_id, torch_dtype=torch.float32)
pipe.scheduler = DDIMScheduler.from_config(pipe.scheduler.config)
pix2pix = StableDiffusionPix2PixZeroPipeline(pipe.vae,pipe.text_encoder,pipe.tokenizer, pipe.unet, pipe.scheduler, pipe.feature_extractor, None, DDIMInverseScheduler.from_config(pipe.scheduler.config), None, None, False)
pipe.to("cuda")
pix2pix.to("cuda");

# some random input
generator = torch.manual_seed(123456)
randomNoise=torch.randn((1,4,64,64))

# generate a image from the given random input
someimg = pipe("A cat sitting on a table", latents=randomNoise, generator=generator, num_inference_steps=50).images[0]
someimg
```
![1](https://user-images.githubusercontent.com/1015048/223282268-7198a96b-9262-4acb-b14c-94789520e1de.png)

```python
# invert the image
result=pix2pix.invert("A cat sitting on a table", someimg, generator=generator, num_inference_steps=50)
result.images[0]
```
![2](https://user-images.githubusercontent.com/1015048/223282310-a0bacbc7-3840-4734-9e88-19b8a3e4d9a5.png)

```python
# as I understand the DDIMInverseScheduler, using this latent input should produce my previous input image again
someimg = pipe("A cat sitting on a table", latents=result.latents, generator=generator, num_inference_steps=50).images[0]
someimg
```
![3](https://user-images.githubusercontent.com/1015048/223282385-e37c294e-5fc4-4f86-9891-2e99898a1467.png)
```python
# but as you can see I just get a blank image back.

# note that this gives me a correct (but of course different) image
someimg = pipe("A cat sitting on a table", latents=torch.randn_like(result.latents), generator=generator, num_inference_steps=50).images[0]
```

### System Info

A matching Triton is not available, some optimizations will not be enabled.
Error caught was: No module named 'triton'

Copy-and-paste the text below in your GitHub issue and FILL OUT the two last points.

- `diffusers` version: 0.14.0
- Platform: Windows-10-10.0.19041-SP0
- Python version: 3.8.5
- PyTorch version (GPU?): 1.13.1 (True)
- Huggingface_hub version: 0.10.1
- Transformers version: 4.26.1
- Accelerate version: 0.16.0
- xFormers version: 0.0.16
- Using GPU in script?: Yes
- Using distributed or parallel set-up in script?: No

**Comment by kaibioinfo** [comment]
If I replace
```python
model_id = "stabilityai/stable-diffusion-2-1"
```
by
```python
model_id = "CompVis/stable-diffusion-v1-4"
```
then the code runs!

So it seems to be a problem with stable diffusion v2-1...

**Comment by clarencechen** [comment]
Bug addressed in #2619

**Comment by sayakpaul** [comment]
Unassigning myself since @clarencechen has already started a PR: https://github.com/huggingface/diffusers/pull/2619

**Comment by sayakpaul** [comment]
Closing this as the issue has been resolved with #2619.

**Comment by pikeyang** [comment]
If I don't want to provide a prompt, can I still complete the inversion process?
